gen3/tools/download/drs_download.py

```diff
@@ -982,6 +998,14 @@ def download(
             res = download_file_from_url(
                 url=download_url, filename=filepath, show_progress=show_progress
             )
+
+            # check if object is a zip file and a package
+            if entry.file_name.split(".")[-1] == "zip":
+                # if so expand in place
+                mds_entry = self.metadata.get(entry.object_id)
+                if mds_entry["type"] == "package":
```

Comment:
```suggestion
                if mds_entry.get("type") == "package":
```
because non-package entries don't currently have a "type"